### PR TITLE
PEO-371 Implement factory_bot runner

### PR DESCRIPTION
**Pull request by D3N**
[PEO-371](https://garage.atlassian.net/browse/PEO-371)

### Description

Need to implement factory_bot runner. 
By default each level of the issue equal low.

### Before submitting the merge request make sure the following are checked

- [x] Followed the style guidelines of this project
- [x] Performed a self-review of own code
- [x] Wrote the tests that prove that fix is effective/that feature works

### CODE REVIEW

File: lib/inquisition/rubocop/runner.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Runner < ::Inquisition::Runner
         fatal: :high
       }.freeze
 
-      ::RuboCop::ConfigLoader::DOTFILE = 'config/.rubocop.yml'.freeze
+      # ::RuboCop::ConfigLoader::DOTFILE = 'config/.rubocop.yml'.freeze
```
Comment (rostikkkk2) [inline lib/inquisition/rubocop/runner.rb]: Why do we need this line?

File: spec/inquisition/factory_bot/runner_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  let(:key_error) { instance_double(KeyError, message: message) }
+  let(:factory) { instance_double(FactoryBot::Factory, name: factory_name) }
+  let(:error) { FactoryBot::Linter::FactoryError.new(key_error, factory) }
+  let(:factory_hash_error) { { factory => [error] } }
+  let(:low_severity_issue) { { severity: :low, path: factory_name, message: "#{factory_name} - #{message}" } }
+
+  describe '#call' do
+    before do
+      allow(runner).to receive(:load_environment).and_return(true)
+      linter = instance_double(FactoryBot::Linter)
```
Comment (rostikkkk2) [inline spec/inquisition/factory_bot/runner_spec.rb]: I think would be better to move `instance_double(FactoryBot::Linter)` in `let()`

File: lib/inquisition/factory_bot/factory_bot_modified_linter.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module Inquisition
+  module FactoryBot
+    class FactoryBotModifiedLinter < ::FactoryBot::Linter
+      def lint!
```
Comment (chubchenko) [inline lib/inquisition/factory_bot/factory_bot_modified_linter.rb]: https://stackoverflow.com/questions/7179016/what-is-the-purpose-of-and-at-the-end-of-method-names

File: spec/inquisition/factory_bot/runner_spec.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
+RSpec.describe Inquisition::FactoryBot::Runner do
+  describe '#call' do
+    let(:runner) { described_class.new }
+    let(:linter) { instance_double(Inquisition::FactoryBot::FactoryBotModifiedLinter) }
```
Comment (chubchenko) [inline spec/inquisition/factory_bot/runner_spec.rb:6]: Do we need all these lets?
Can we do something like this:

```ruby
let(:linter) { instance_double(Inquisition::FactoryBot::FactoryBotModifiedLinter) }
let(:factory_error) { instance_double(FactoryBot .... FactoryError }
let(:factory_trait_error) { instance_double(FactoryBot .... FactoryTraitError }

before do
  allow(linter).to receive(....).and_return(
    [factory_error, factory_trait_error]
  )
end

it '...' do
  expect(runner.call).to contain_exactly(...)
end
```

what do you think?

File: spec/inquisition/factory_bot/factory_bot_modified_linter_spec.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+RSpec.describe Inquisition::FactoryBot::FactoryBotModifiedLinter do
+  describe '#lint!' do
+    let(:error) { instance_double(FactoryBot::Linter::FactoryError) }
```
Comment (chubchenko) [inline spec/inquisition/factory_bot/factory_bot_modified_linter_spec.rb]: Propose to discuss it

**Review by chubchenko — CHANGES_REQUESTED** [review]

File: lib/inquisition/factory_bot/runner.rb [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+      def call
+        load_environment
+        errors = FactoryBotModifiedLinter.new(::FactoryBot.factories, traits: true).perform.values.flatten
+        errors.each { |error| create_issues(error) }
+        @issues
+      end
+
+      private
+
+      def load_environment
```
Comment (Svatok) [inline lib/inquisition/factory_bot/runner.rb:15]: We have this methods in several runners. Maybe we should move this method to `Inquisition::Runner`? What do you think about this?

**Pull request merged**